docs/In-Band Manageability User Guide - Azure.md

```diff
@@ -1106,6 +1106,38 @@ In order to trigger Decommission:
     <img src="media/In-Band Manageability User Guide - Azure/media/image65.png" style="width:3.02153in;height:1.91736in" />
 
 ### Query Commands
+Description
+
+The Query command can be called by either the **cloud or INBC**. It will provide attribute information on either the Host, Edge Device, or Nodes.
+
+How to Call
+
+•[INBC](https://github.com/intel/intel-inb-manageability/blob/develop/inbc-program/README.md#Query)
+
+To trigger a **Trigger Query**, follow the steps below:
+
+In order to query updates, click the **Trigger Query button** as seen below.
+
+<img src="https://github.com/intel/intel-inb-manageability/blob/RTC_Fixed_Branch_497530/docs/media/In-Band%20Manageability%20User%20Guide%20-%20Azure/media/image87.PNG" style="width:5.78125in;height:3.59762in" />
+
+Populate the **Trigger Query** pop-up window with the required parameters and click **run** to trigger the query Update.
+
+<img src="https://github.com/intel/intel-inb-manageability/blob/RTC_Fixed_Branch_497530/docs/media/In-Band%20Manageability%20User%20Guide%20-%20Azure/media/image88.PNG" style="width:5.78125in;height:3.59762in" />
+
+**Find the Query Option and the Description below:**
+
+
```

Comment:
Since we already have a document that has the options, we don't want to have another place to change in the event we do make changes in the future.  We should just reference where we have that information. Which is the query.md file in the same directory as this document.